.env.test

```diff
@@ -1,2 +1,3 @@
 SERVER_HOST=localhost
 PASSWORD_RESET_URL=/
+TEST_ENV_NUMBER=3
```

Comment:
I don't think this should be set 🤔

Reply:
I just added as a part of the first step of adding this gem, I can delete it if you want

Reply:
but it doesn't make sense to have it here, or why do you want to have it?